src/main/java/io/jenkins/plugins/jfr/JfrAction.java

```diff
@@ -0,0 +1,32 @@
+package io.jenkins.plugins.jfr;
+
+import hudson.model.Action;
+import java.util.Collection;
+import java.util.Collections;
+import org.jspecify.annotations.NonNull;
+
+public class JfrAction implements Action {
```

Comment:
Please update this class to be named `JavaFlightRecorderAction` for clarity. It should also be a `RootAction` annotated with `@Extension`.